shipyard-controller/internal/db/mongodb_dbdump_repo_test.go

```diff
@@ -0,0 +1,18 @@
+package db_test
+
+import (
+	"testing"
+
+	"github.com/keptn/keptn/shipyard-controller/internal/db"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMongoDBDumpRepo_GetDump(t *testing.T) {
```

Comment:
you are testing two endpoints in the same test case. I would split it so it is easier to get the root function that fails